src/assets/strings/es.ts

```diff
@@ -52,6 +52,7 @@ const es = {
   transcript: "Transcripción",
   agenda: "Agenda",
   session: "Sesión",
+  passed: "Passed",
```

Comment:
Add "Failed"

Add "Passed" and "Failed" for German too

Add `passed: string` and `failed: string` to src/assets/LocalisedString.tsx